jaxrs-tck/docs/assertions/JAXRSSpecAssertions_3.1.0.html

```diff
@@ -1216,7 +1216,7 @@ <h2>  Jakarta RESTful Web Services - 3.1<br>
 </TR>
 
 <TR>
-<TD align="center" valign="center"><a name="JAXRS:SPEC:131"></a><font size="1PT">JAXRS:SPEC:131</font></TD><TD align="center" valign="center"><font size="1PT">2</font></TD><TD align="center" valign="center"><font size="1PT">3.1</font></TD><TD align="left" valign="center"><font size="1PT">There are two alternative ways of publishing on Java SE: Creating SE endpoints directly or using the SE bootstrap API. Any compliant implementation that supports either of these alternatives on Java SE MUST support both. </font></TD><TD align="center" valign="center"><font size="1PT">true</font></TD><TD align="center" valign="center"><font size="1PT">
+<TD align="center" valign="center"><a name="JAXRS:SPEC:131"></a><font size="1PT">JAXRS:SPEC:131</font></TD><TD align="center" valign="center"><font size="1PT">2</font></TD><TD align="center" valign="center"><font size="1PT">3.1</font></TD><TD align="left" valign="center"><font size="1PT">There are two alternative ways of publishing on Java SE: Creating SE endpoints directly or using the SE bootstrap API. A compliant implementation MUST support both alternatives on Java SE. </font></TD><TD align="center" valign="center"><font size="1PT">true</font></TD><TD align="center" valign="center"><font size="1PT">
```

Comment:
This is not a correct change.   It was changed recently in [PR 1231](https://github.com/jakartaee/rest/pull/1231), please remove this change

Reply:
I assume the changes in PR1231 is required for release 4.0, hence added the same in new file jaxrs-tck/docs/assertions/JAXRSSpecAssertions_4.0.0.html. Is that ok.

Reply:
Yes,   while it is an appropriate change for 3.1, it doesn't need to be put there.